dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -282,26 +287,29 @@ void CDefaultContextMenu::AddStaticEntry(const HKEY hkeyClass, const WCHAR *szVe
         pEntry->hkClass = hkeyClass;
     }
 
-    if (!wcsicmp(szVerb, L"open"))
+    lres = RegQueryValueEx(hShellKey, NULL, 0, &pdwType, (LPBYTE)wszDefault, &dwSize);
```

Comment:
Should check whether the RegOpenKey call succeeded first, or hShellKey might be invalid.
Should also call the -W version explicitly.